bravado/mapping/schema.py

```diff
@@ -0,0 +1,82 @@
+from bravado.exception import SwaggerError
+from bravado import swagger_type
+
+
+JSONSCHEMA_PRIMITIVE_VALIDATIONS = {
+    'string': ('minLength', 'maxLength', 'pattern', 'format', 'enum', 'default'),
+    'integer': ('multipleOf', 'minimum', 'maximum', 'exclusiveMaximum', 'exclusiveMinimum', 'enum', 'default', 'format'),
```

Comment:
i think it will exceed 80. Also, is there really a need to pick these out separately for each type? Doesnt jsonschema just ignore the fields not applicable?